crud-app/database/migrations/2023_03_15_044227_add_name_to_reviews.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('reviews', function (Blueprint $table) {
+            $table->string('name')->index();
```

Comment:
do not have this on a review. if you are seeking the name of the user who left the review then get that from the relationship: review->user->name

Reply:
i tried to rollback my migration to remove this, and when that wasn't working out, I attempted to revert all my changes. Now my code is up to date with the latest push on this branch, however my Laravel is acting up and I can no longer access my review page in browser because columns can't be found.